.github/workflows/centaur_integration_tests.yml

```diff
@@ -0,0 +1,61 @@
+name: 'Centaur Integration Tests'
+
+#This github action runs all of Cromwell's integration tests.
+
+#This is what shows up in the github workflows page as the title. 
+run-name: ${{ github.actor }} Centaur Testing.
```

Comment:
These aren't all Centaur test, right? Maybe should say Integration Testing instead?